Cluster/How to Setup Azure Alerts for Service Fabric Node Performance Counters.md

```diff
@@ -0,0 +1,460 @@
++                                                            "sampleRate": "PT10S"
++                                                        }
++                                                    ]
++                                                },
+                                                 "EtwProviders": {
+                                                     "EtwEventSourceProviderConfiguration": [
+                                                         {
+```
+
+## Add sink
+
+Starting with Diagnostics extension version 1.5, 'sinks' has been added to implement a sink location to send diagnostic data. Using 'sink' and Custom Metrics, performance counter data can be uploaded and viewed in Azure Portal. 'SinksConfig' element provides a list of locations to send diagnostics data to. Examples are, 'AzureMonitorSink', 'ApplicationInsights', and 'EventHub'. In this example, 'AzureMonitor' is being used. Empty 'resourceId' is required.
+
+```diff
+diff --git a/template/template.json b/template/template.json
+index a249f60..469dbf0 100644
+--- a/template/template.json
++++ b/template/template.json
+@@ -502,6 +502,7 @@
+                                                 "overallQuotaInMB": "50000",
+                                                 "PerformanceCounters": {
+                                                     "scheduledTransferPeriod": "PT1M",
++                                                    "sinks": "AzureMonitorSink",
+                                                     "PerformanceCounterConfiguration": [
+                                                         {
+@@ -545,6 +546,14 @@
+                                                         }
+                                                     ]
+                                                 }
++                                            },
++                                            "SinksConfig": {
++                                                "Sink": [
++                                                    {
++                                                        "name": "AzureMonitorSink",
++                                                        "AzureMonitor": {
++                                                            "resourceId": ""
++                                                        }
++                                                    }
++                                                ]
+                                             }
+                                         },
+                                         "StorageAccount": "[parameters('applicationDiagnosticsStorageAccountName')]"
+```
+
+## Updating deployment configuration
+
+If using a template to modify configuration, after .\template.json has been modified, run 'test-azResourceGroupDeployment' and 'new-azResourceGroupDeployment' to deploy new template.
```

Comment:
For readability: Test-AzResourceGroupDeployment, New-AzResourceGroupDeployment

Reply:
fixed